### ISSUE TITLE
Incorrect style "TransNav" v21

### ISSUE THREAD

**Issue by SergeyAppDroid** [roughike/BottomBar#747]
I had a problem that is described here: [https://github.com/roughike/BottomBar/issues/737](url)

I was prompted by such a decision:
[https://github.com/roughike/BottomBar#how-do-i-draw-it-under-the-navbar](url)

But this solution does not work right! 
Android 4 (nexus 5):
![device-2017-04-11-231021](https://cloud.githubusercontent.com/assets/26580188/24926260/1694a5f0-1f0c-11e7-97b7-860702e38a65.png)

Android 5 (nexus 5X) after using style v21:
![device-2017-04-11-231304](https://cloud.githubusercontent.com/assets/26580188/24926458/aff0853e-1f0c-11e7-9a3f-f7af5a7204e7.png)

Original picture:
![1](https://cloud.githubusercontent.com/assets/26580188/24926422/87f89256-1f0c-11e7-89d2-e3118903c17d.png)

**Comment by yombunker** [comment]
@SergeyAppDroid do you have the shy mode as well? or just undernavbar?

**Comment by SergeyAppDroid** [comment]
use shy|underNavbar:

`<com.roughike.bottombar.BottomBar
            android:id="@+id/bottomBar"
            android:layout_width="match_parent"
            android:layout_height="@dimen/bottomBarSize"
            android:layout_alignParentBottom="true"
            android:layout_gravity="bottom"
            android:background="@color/colorPrimary"
            app:bb_behavior="shy|underNavbar"
            app:bb_tabXmlResource="@xml/bottombar_tabs"
            app:bb_inActiveTabAlpha="0.6"
            app:bb_activeTabAlpha="1"
            app:bb_activeTabColor="@color/icons"
            app:bb_showShadow="false"
            app:bb_inActiveTabColor="@color/colorInActive"
            app:bb_badgesHideWhenActive="true"/>`

**Comment by yombunker** [comment]
yes, so the shy behavior if i'm correct @roughike can correct me if i'm wrong, what it does is that it allows the content to be drawn underneath so that when you scroll and the bottobar hides, you can still see the content behind it without having to resize as you scroll. And that's why you are getting that behavior, if you don't want that then remove shy, and if you do want shy, then you need to add the margins to the imageview so that it only uses the visible area, or use the anchor attribute of the coordinatorlayout, hope this helps

**Comment by SergeyAppDroid** [comment]
Now I do this:
1) determine the height of the NavigationBar for a particular device
2)` imageView.setPadding(0,0,0,navBarHeight)`;

It works, but can there be a more correct way?
